charts/stable/privatebin/templates/common.yaml

```
@@ -0,0 +1,16 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Append the configMap volume to the volumes */}}
+{{- define "privatebin.configVolume" -}}
+enabled: "true"
+mountPath: "/srv/cfg/conf.php"
+subPath: "conf.php"
+type: "custom"
+volumeSpec:
+  configMap:
+    name: {{ template "common.names.fullname" . }}-config
+{{- end -}}
+{{- $_ := set .Values.persistence "privatebin-config" (include "privatebin.configVolume" . | fromYaml) -}}
```

```suggestion
{{/* Append the hardcoded settings */}}
{{- define "privatebin.harcodedValues" -}}
  {{/* Append the configMap volume to the volumes */}}
  persistence:
    privatebin-config:
      enabled: true
      type: "configMap"
      name: "{{ include "common.names.fullname" . }}-config"
      mountPath: "/srv/cfg/conf.php"
      subPath: "conf.php"
  {{- end }}
{{- end -}}
{{- $_ := mergeOverwrite .Values (include "privatebin.harcodedValues" . | fromYaml) -}}
```

We have (sort of unofficially) moved away from the previous style of injecting "hardcoded" values and moved to the pattern as I've laid out in my suggestion.